### PR TITLE
ADD BCP Coin

### PR DESCRIPTION
**Pull request by JoshepJang**
I want to show you the symbolic image erc-20 token in the metamask.

### CODE REVIEW

**Comment by xrn** [comment]
@JoshepJang can you solve conflicts and upload smaller versions of svg images - current size over 20kB for both is pretty high.

**Comment by JoshepJang** [comment]
> @JoshepJang 은 충돌을 해결하고 더 작은 버전의 svg 이미지를 업로드할 수 있습니다. 둘 다 현재 크기가 20kB를 초과하면 꽤 높습니다.



> @JoshepJang can you solve conflicts and upload smaller versions of svg images - current size over 20kB for both is pretty high.

I've reduced the size. Thank you.

**Comment by xrn** [comment]
Hi @JoshepJang - state of PR is incorrect.

1. You are using svg in JSON and you have 2 svg added but also you added 3rd png which is not used
2. This repo has requirement that icons has to be in SVG - please remove PNG one and reduce size of SVG - thanks in advance

**Comment by JoshepJang** [comment]
> Hi @JoshepJang - state of PR is incorrect.
> 
> 1. You are using svg in JSON and you have 2 svg added but also you added 3rd png which is not used
> 2. This repo has requirement that icons has to be in SVG - please remove PNG one and reduce size of SVG - thanks in advance

I've processed your request.
Do I need to reduce the size more?

**Comment by xrn** [comment]
@JoshepJang - Camp.svg  is still over 70kB - In my opinion you are generating SVG in wrong way. This are PNG files embedded into PNG and not curves which are used by correct SVG. If you can reduce size of Camp.svg  will be great and I would also advice to use correct SVG images which can be easily resized.

**Comment by JoshepJang** [comment]
> @JoshepJang - Camp.svg is still over 70kB - In my opinion you are generating SVG in wrong way. This are PNG files embedded into PNG and not curves which are used by correct SVG. If you can reduce size of Camp.svg will be great and I would also advice to use correct SVG images which can be easily resized.

Sorry, CAMP is already registered. 
All you have to do is register for BCP.
Do I need to modify the CAMP image?

**Comment by xrn** [comment]
@JoshepJang if CAMP is added please remove it from this PR and add only BCP

**Comment by JoshepJang** [comment]
> @JoshepJang if CAMP is added please remove it from this PR and add only BCP

I deleted the camp and uploaded a reduced size.

File: contract-map.json [hunk 1]
```diff
@@ -1976,5 +1976,19 @@
     "erc20": true,
     "symbol": "NEXO",
     "decimals": 18
+  },
+  "0xE9E73E1aE76D17A16cC53E3e87a9a7dA78834d37": {
```
Comment (xrn) [inline contract-map.json:1980]: Please remove also CAMP from JSON in this place because it is duplication and resolve conflicts against master

Comment (JoshepJang) [reply contract-map.json]: I've processed your confirmation.

**Comment by xrn** [comment]
Please remove COMP.svg file from this PR or replace old one and resolve conflicts in contract-map.json against master

**Review by xrn — APPROVED** [review]

**Pull request merged**